manifests/init.pp

```diff
@@ -35,6 +35,7 @@
   $password_max_age    = 90,
   $shell               = '/bin/bash',
   $home                = "/home/${name}",
+  $managehome          = true,
```

Comment:
It originally hard-coded to true in your manifest, so I was matching
that. You're right about it defaulting to false in the built-in type,
though. So set it to false then?

On 12/18/2015 04:06 PM, Rob Nelson wrote:
> In manifests/init.pp
> <https://github.com/rnelson0/puppet-local_user/pull/18#discussion_r48070020>:
> 
>> @@ -35,6 +35,7 @@
>>    $password_max_age    = 90,
>>    $shell               = '/bin/bash',
>>    $home                = "/home/${name}",
>> +  $managehome          = true,
> 
> This defaults to false in the user resource, so should here as well.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rnelson0/puppet-local_user/pull/18/files#r48070020>.
>